tests/parse_ballot.py

```diff
@@ -0,0 +1,96 @@
+import sys
```

Comment:
This is the class used to generate the TestScenario classes. Generating php via python? Gross, I know. See PR description for justification of this technical debt.